portfolio/src/main/webapp/index.html

```diff
@@ -5,21 +5,35 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+    body {
+      background-image: url('https://i.pinimg.com/originals/92/17/4a/92174ad3758fdb8f9c2e4579ad5430a0.jpg');
+      background-size: cover;
+    }
+    </style> -->
   </head>
   <body>
+    <div class="header" style="background-image: url('https://wallpaperaccess.com/full/1425559.jpg'); background-size: cover;">
+    <a href="#default" class="logo"><img src=/images/logo.png alt="My logo" width="70" height="70">
+    <div class="header-right">
+        <a class="active" href="#home">Home</a>
+        <a href="#contact">Contact</a>
+        <a href="#about">About</a>
+    </div>
+    </div>
+    <h1 class="h1 center">Student, Tech Enthusiast & Dreamer</h1>
```

Comment:
Maybe you want to use an h1 element selector instead of an "h1" class selector: https://www.w3schools.com/css/css_selectors.asp

Given that this is an h1 element it seems unnecessary to create a css class "h1". Otherwise maybe you just want to create one class that combines the styles you have for h1 + center into one css class.